railties/lib/rails/generators/rails/plugin/templates/Gemfile.tt

```diff
@@ -15,7 +15,7 @@ gem "rubocop-rails-omakase", require: false
 <% if RUBY_ENGINE == "ruby" -%>
 
 # Start debugger with binding.b [https://github.com/ruby/debug]
-# gem "debug", ">= 1.0.0"
+gem "debug", ">= 1.0.0"
```

Comment:
This is the only commented out gem in plugins, and [it isn't commented out in apps](https://github.com/rails/rails/blob/aa66aece44f191e6330c04ce4942c3edf31d85d7/railties/lib/rails/generators/rails/app/templates/Gemfile.tt#L56), so I opted to un-comment it here rather than have a separate conditional initializer just for plugins that does something hacky like check the loaded gems first.